chain/syncer_integration_test.go

```diff
@@ -397,6 +398,10 @@ func TestLoadFork(t *testing.T) {
 
 	// Shut down store, reload and wire to syncer.
 	loadSyncer, blockSource := loadSyncerFromRepo(t, r, dstP)
+	// The fetcher will need to fetch these so that is may pass them to the
+	// done callback inorder to determine when to stop fetching.
+	// DONOTMERGE get specific approval on this before merging
+	blockSource.AddSourceBlocks(forklink3.ToSlice()...)
```

Comment:
The fetcher needs this value in its store as it uses it to determine when it can stop fetching.